chrome/content/zotero/preferences/preferences_export.js

```diff
@@ -133,35 +139,55 @@ Zotero_Preferences.Export = {
 	},
 	
 	
-	updateQuickCopyHTMLCheckbox: function () {
+	updateQuickCopyUI: function () {
 		var format = document.getElementById('zotero-quickCopy-menu').value;
+		
 		var mode, contentType;
 		
-		var checkbox = document.getElementById('zotero-quickCopy-copyAsHTML');
 		[mode, format] = format.split('=');
 		[mode, contentType] = mode.split('/');
 		
+		var checkbox = document.getElementById('zotero-quickCopy-copyAsHTML');
 		checkbox.checked = contentType == 'html';
 		checkbox.disabled = mode != 'bibliography';
+		
+		var menulist = document.getElementById('locale-menu');
+		if (mode != 'bibliography') {
+			// maintain menu selection but show blank label
+			menulist.insertItemAt(0, '', menulist.value);
+			menulist.selectedIndex = 0;
+			menulist.disabled = true;
```

Comment:
It wasn't very hard to do this for the Export tab of the preferences window (see https://github.com/rmzelle/zotero/commit/70e96b9a009d990c728e5f2b57f37c700fb3290f ), but I don't know how to do this for the site-specific settings window, where the label is displayed via a `<treecol/>` element, which can't be disabled (`<treecol id="quickCopy-localeColumn" label="&zotero.preferences.quickCopy.siteEditor.locale;"/>` in preferences_export_firefox.xul)